addons/l10n_mt/data/template/account.tax-mt.csv

```diff
@@ -1,89 +1,111 @@
 "id","sequence","description","invoice_label","name","price_include","amount","amount_type","type_tax_use","tax_group_id","repartition_line_ids/factor_percent","repartition_line_ids/repartition_type","repartition_line_ids/document_type","repartition_line_ids/tag_ids","repartition_line_ids/account_id"
 "VAT_S_IN_MT_18_G","1","Sale 18% in country Goods","18% G","18% G","False","18.0","percent","sale","tax_group_mt_18","100","base","invoice","",""
+"","","","","","","","","","","100","base","invoice","+III.1_base",""
```

Comment:
That's not correct. It's a bit counter-intuitive but tax in CSV definition are in most cases 4 lines.
The first one contains the general tax info + the base-invoice tax line.
TLDR; you need to put the tag in the line above :slightly_smiling_face: 

The rest LGTM :+1:

Reply:
Yes :slightly_smiling_face: